.github/workflows/on-qa-firebase-distribution.yml

```diff
@@ -38,6 +36,8 @@ jobs:
         env:
             # API URL for the environment selected
             ORG_GRADLE_PROJECT_API_URL: ${{ secrets.API_URL }}
+            # Claim DApp URL
+            ORG_GRADLE_PROJECT_CLAIM_APP_URL: ${{ secrets.CLAIM_APP_URL }}
```

Comment:
Is it worth declaring this claim app URL as input with 2 options (`claim` and `claim-app-dev`)?

Reply:
Hmm I don't think we need it, as we are playing with environment secrets. It's definitely easier having two different options (with dev suffix) but we had structured already the building process to use only one variable (as in `API_URL` also) exactly for that reason, to not have multiple variables for the same thing.